third_party/istio-1.5.7/install-istio.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Download and unpack Istio
+ISTIO_VERSION=1.5.7
+ISTIO_TARBALL=istio-${ISTIO_VERSION}-linux.tar.gz
+DOWNLOAD_URL=https://github.com/istio/istio/releases/download/${ISTIO_VERSION}/${ISTIO_TARBALL}
+
+wget --no-check-certificate $DOWNLOAD_URL
```

Comment:
The `istio-1.6.5-linux-amd64.tar.gz` is around 40M and it takes 5 to 10 min (on my local).
It is alright to download the tar file every single e2e tests?
As you might known, in the serving repo I generated the manifest by `istioctl manifest generate` to avoid it.

Reply:
Fair remark for slow internet connections and local development.

You can see in the logs that in the CI system takes ~1s to download the package.
My concern with `istioctl generate manifest` + `kubectl apply` is that unlike `istioctl install` it doesn't have ordering guarantees and is actually not recommended. See https://istio.io/latest/docs/setup/install/istioctl/#generate-a-manifest-before-installation.

> The output from manifest generate can also be used to install Istio using kubectl apply or equivalent. However, these alternative installation methods may not apply the resources with the same sequencing of dependencies as istioctl install and are not tested in an Istio release.

Reply:
But 5 to 10 minutes is insane. That's like 60 kBps 😢. Is there an issue with Github and Japan somehow?

Reply:
I see. Yeah, then there are some issue in Github or Japan :smile: